.gitignore

```diff
@@ -0,0 +1,20 @@
+*.annot
+*.cmo
+*.cma
+*.cmi
+*.a
+*.o
+*.cmx
+*.cmxs
+*.cmxa
+
+# ocamlbuild working directory
+_build/
+
+# ocamlbuild targets
+*.byte
+*.native
+
+# oasis generated files
+setup.data
+setup.log
```

Comment:
I think the `.gitignore` could be simplified because `_build/` ignores already the files listed above. However, there is nothing wrong here and I'm happy to keep it as is.